examples/example-evals-nextjs/src/lib/capabilities/classify-ticket/evaluations/ticket-classification.eval.ts

```diff
@@ -19,7 +19,8 @@ const WrappedExactMatch = Scorer(
 
 Eval('spam-classification', {
   configFlags: pickFlags('ticketClassification'),
-  capability: 'classify-ticket',
+  capability: 'ticket-classification',
```

Comment:
I named it `classify-ticket` because that's how we named the directory above: `src/lib/capabilities/classify-ticket`.